website/content/docs/intro/use-cases/migrate-clouds.mdx

```diff
@@ -0,0 +1,258 @@
+layout: docs
+page_title: Seamless Cloud Migration With Waypoint
+description: |-
+  Waypoint enables easy migration of applications between different cloud providers.
+---
+
+# Seamless Cloud Migration With Waypoint
+
+A time may come where you need to move your application from one cloud provider to
+another, or one data center to another, but doing so can be cumbersome. There may be
+no standards between application tools for building, you may have special scripts for
+deploying, and releasing may prove to be difficult since you can't release across
+platforms in the same manner. Waypoint can solve those problems. With Waypoint, it is
+as easy as deploying a runner to your new cluster, changing kubectl context, and
+running `waypoint up` to migrate your application across multiple cloud providers.
+
+This use case starts with an application deployed in AWS EKS and showcases migrating
+that same application to GCP GKE, and then Azure AKS.
+
+## Pre-Requisites
+
+If experimenting with this use case, it will be helpful to have some familiarity
+with the tools and concepts listed below.
+
+- [Waypoint CLI](https://developer.hashicorp.com/waypoint/downloads)
+  - [Context created](https://developer.hashicorp.com/waypoint/docs/server/auth#logging-in) and connected to local Waypoint server or HCP Waypoint server
+- Waypoint [runner](https://developer.hashicorp.com/waypoint/docs/runner/run-manual#installing-a-runner)
+  - Must be able to connect to your Kubernetes clusters
+- Waypoint [runner configuration](https://developer.hashicorp.com/waypoint/docs/runner/config)
+  - Needs credentials to your registry
+- Docker registry
+  - Credentials to push a build to the registry
+- [AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/clusters.html) cluster
+- [GCP GKE](https://cloud.google.com/kubernetes-engine) cluster
+- [Azure AKS](https://azure.microsoft.com/en-us/products/kubernetes-service/) cluster
+
+## Migrate from AWS EKS to GCP GKE
+
+##### Step 1. Start with an existing Kubernetes cluster and application deployed
+
+##### Step 2. Configure your kubectl context to point to your new cluster you are migrating to
+
+```shell
+# Use the context of your cluster you would like to migrate to
+$ kubectl config use-context gcp-context
+```
+
+##### Step 3. Waypoint runner install into your new cluster
+
+Install a runner into your cluster you are migrating to and configure the runner
+profile with your registry credentials.
+
+```shell
+$ waypoint runner install -platform=kubernetes -server-addr=<server_addr>  -k8s-runner-image=hashicorp/waypoint:latest -id=gcp -- -label=cloud=gcp
+✓ Finished connecting to: <server_addr>
+⠧ Installing runner...
+✓ Finished connecting to: <server_addr>
+✓ Runner "gcp" installed successfully to kubernetes
+✓ Runner profile "kubernetes-gcp" created successfully.
+✓ Waypoint runner installed with Helm!
+✓ Runner "gcp" adopted successfully.
+
+$ waypoint runner profile set -env-var=DOCKER_PWD='pwd' -name=kubernetes-gcp
```

Comment:
You are correct that the runner install does create the runner + runner profile. I ran into an err due to the runner not having proper credentials for authentication to build and push to my Docker registry, which is why that `waypoint runner profile set` command is needed. 

Could you clarify what you mean by `we might want to specify changing the default runner profile`? Im not sure that I follow. I thought by adding that explicit command in then it would trip less people up, as I myself got tripped up not realizing my runner profile needed credentials.

Reply:
> You are correct that the runner install does create the runner + runner profile. I ran into an err due to the runner not having proper credentials for authentication to build and push to my Docker registry, which is why that waypoint runner profile set command is needed.

I'd be curious to see what bug this was! Setting that env var shouldn't be required to configure registry credentials. `DOCKER_PWD` sets the working directory for docker if read by the docker engine. Kaniko should be doing the build + registry push in an ODR and I don't think it reads that env var. I've never had to set that env var to get registry pushing working and I don't think it's required for our ODR profiles. If there's something missing or broken in the runner profile set up by `waypoint runner install` we should fix it though!

> Could you clarify what you mean by we might want to specify changing the default runner profile? Im not sure that I follow. I thought by adding that explicit command in then it would trip less people up, as I myself got tripped up not realizing my runner profile needed credentials.

It was a suggestion to briefly explain in the guide that `waypoint runner install` has installed a runner profile named `kubernetes-gcp` but did't change your "default" profile if you had one already. Perhaps too much info at this point though. Since your guide ends up configuring the app to use the specific profile, so that suggestion might be extra noise.

Reply:
I think I may have overloaded that term. I didn't realize `DOCKER_PWD` was something docker expected for the print working dir. I was using that locally as an env var for my docker access token credentials. I overloaded pwd as a shorthand for password. As an alternative, I can change the env var name to be more explicit.

I ran into an unauthorized err where my ODR didn't have proper access to push to the registry without the credential being set. After discussing this, I will update our docs to add a more explicit note that for env vars you must run the `waypoint runner profile set -env-var=key=val` for setting the env var on a specific runner profile or use the `waypoint config set -runner` for the runner + odr to have proper creds.

Reply:
The `DOCKER_PWD`  is now `DOCKER_PASSWORD`. As well as, the `waypoint runner profile set` command is no longer needed now that I know about the `waypoint config set -runner` command.